fabric-screen-api-v1/src/client/java/net/fabricmc/fabric/mixin/screen/ScreenMixin.java

```diff
@@ -150,7 +150,7 @@ public List<ClickableWidget> fabric_getButtons() {
 	@Unique
 	private <T> Event<T> ensureEventsAreInitialised(Event<T> event) {
 		if (event == null) {
-			throw new IllegalStateException(String.format("[fabric-screen-api-v1] The current screen (%s) has not been correctly initialised, please send this crash log to the mod author. This is usually caused by the screen not calling super.init(Lnet/minecraft/client/MinecraftClient;II)V", this.getClass().getSuperclass().getName()));
+			throw new IllegalStateException(String.format("[fabric-screen-api-v1] The current screen (%s) has not been correctly initialised, please send this crash log to the mod author. This is usually caused by calling setScreen on the wrong thread.", this.getClass().getSuperclass().getName()));
```

Comment:
Have you got an example of this beign the case? I think the .getSuperclass() needs here though?

Reply:
Think so. And yes, reaching this is as easy as `new Thread(() -> MinecraftClient.getInstance().setScreen(screen)).start()`

Reply:
```suggestion
			throw new IllegalStateException(String.format("[fabric-screen-api-v1] The current screen (%s) has not been correctly initialised, please send this crash log to the mod author. This is usually caused by calling setScreen on the wrong thread.", this.getClass().getName()));
```